addons/website_sale/tests/test_website_sale_pricelist.py

```diff
@@ -36,7 +36,8 @@ def setUp(self):
         self.website = self.env.ref('website.default_website')
         self.website.user_id = self.env.user
 
-        (self.env['product.pricelist'].search([]) - self.env.ref('product.list0')).write({'website_id': False, 'active': False})
+        self.env['product.pricelist'].search([]).action_archive()
+        self.env['product.pricelist'].create({'name': 'Public Pricelist'})
```

Comment:
Don't you need to set the website_id to keep the old behavior? :thinking:

Reply:
Creating a pricelist with the `website_sale` module installed will use the default website unless specified not to. So it's not needed ^^